lib/elixir_sense.ex

```diff
@@ -75,15 +76,15 @@ defmodule ElixirSense do
       ...>   MyMod.function_arity_one("some string")
       ...> end
       ...> '''
-      iex>  %{found: true, file: path, line: line, column: column} = ElixirSense.definition(code, 3, 11)
+      iex>  %{file: path, line: line, column: column} = ElixirSense.definition(code, 3, 11)
       iex> "#{Path.basename(path)}:#{to_string(line)}:#{to_string(column)}"
       "module_with_functions.ex:6:7"
   """
-  @spec definition(String.t(), pos_integer, pos_integer) :: Location.t()
+  @spec definition(String.t(), pos_integer, pos_integer) :: Location.t() | nil
   def definition(code, line, column) do
     case Source.subject(code, line, column) do
       nil ->
-        %Location{found: false}
```

Comment:
@lukaszsamson what do we need to change in ElixirLS to allow this change to work?

[This line](https://github.com/elixir-lsp/elixir-ls/blob/master/apps/language_server/lib/language_server/providers/definition.ex#L11-L12) is triggering an error with `unknown key :found for struct ElixirSense.Location`

Reply:
Here basically handling `nil` instead of `%Location{found: false}` but there are other changes. I have it on my local branch already, PR should be ready soon

Reply:
https://github.com/elixir-lsp/elixir-ls/pull/415